tests/SprykerEcoTest/Zed/Adyen/Business/Facade/AdyenFacadeExecutePostSaveHookTest.php

```diff
@@ -136,9 +188,13 @@ public function testExecutePostSaveHookShouldNotContainErrorsWhenResponsIsSucces
     {
```

Comment:
The not clear expectation in the method name `ShouldNotContainErrors`, but then we expect errors in `$this->assertCount(1, $checkoutResponseTransfer->getErrors());` and check them.